### PR TITLE
fix edit and delete tag

### PR DESCRIPTION
**Pull request by Firebreak-2**
h

### CODE REVIEW

**Review by MaxTechnics — APPROVED** [review]
yes?

File: src/commands/Tags/deletetag.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ module.exports = {
         const tag = await bot.tags.findOne({ where: { name: args[0] } });
         if (!tag) return message.channel.send(`The tag \`${args[0]}\` doesn't exist.`);
 
-        if (!await checkStaff(message.member) || tag.username !== message.author.username) return message.channel.send('You can\'t delete this tag.');
+        if (tag.username !== message.author.username && !await checkStaff(message.member)) return message.channel.send('You can\'t edit this tag.');
```
Comment (TeraBaito) [inline src/commands/Tags/deletetag.js]:
```suggestion
        if (tag.username !== message.author.username && !await checkStaff(message.member)) return message.channel.send('You can\'t delete this tag.');
```

**Review by TeraBaito — CHANGES_REQUESTED** [review]
this rq

**Review by TeraBaito — APPROVED** [review]

**Comment by MaxTechnics** [comment]
> 
> 
> this rq

how did i miss this

**Pull request merged**